src/nvim/menu.c

```diff
@@ -276,8 +277,8 @@ add_menu_path (
   char_u      *dname;
   char_u      *next_name;
   int i;
-  int c;
-  int d;
+  char_u c;
+  char_u d;
```

Comment:
While you are at it, you could move this declaration down to line 425, where c and d are initalized. They don't seem to have a broader scope. Also a more descriptive name than a monoletter variable would be great, but maybe moving the declaration scope will be sufficent to make clear what they do.